.prettierrc

```diff
@@ -1,6 +1,5 @@
 {
   "singleQuote": true,
   "trailingComma": "es5",
-  "tabWidth": 2,
-  "plugins": ["./node_modules/prettier-plugin-jsdoc/dist/index.js"]
```

Comment:
please don't remove this plugin...

Reply:
It got pushed by husky.
I'm still testing things.
btw, can we get together so we can sort this thing out please?
I've been suffering with this for weeks.